src/jupyter-datascience-notebooks/devcontainer-template.json

```diff
@@ -0,0 +1,41 @@
+{
+	"id": "jupyter-datascience-notebooks",
+	"version": "1.0.0",
+	"name": "Jupyter Data Science Notebooks",
+	"description": "Use Jupyter Data Science Notebooks with Python, R, Julia, and more.",
+	"documentationURL": "https://github.com/devcontainers-community/templates/tree/main/src/jupyter-datascience-notebooks",
+	"publisher": "Community",
+	"licenseURL": "https://github.com/devcontainers-community/templates/blob/main/LICENSE",
+	"options": {
+		"baseVariant": {
+			"type": "string",
+			"description": "Base image. Check the documentation for details.",
+			"default": "datascience-notebook",
+			"proposals": [
+				"base-notebook",
+				"minimal-notebook",
+				"r-notebook",
+				"scipy-notebook",
+				"tensorflow-notebook",
+				"datascience-notebook",
+				"pyspark-notebook",
+				"all-spark-notebook"
+			]
+		},
+		"imageVariant": {
+			"type": "string",
+			"description": "Image version (tag).",
+			"default": "latest",
+			"proposals": ["latest"]
+		}
+	},
+	"platforms": [
+		"Python",
+		"R",
+		"Julia",
+		"numpy",
+		"pandas",
+		"tidyverse",
+		"much more"
```

Comment:
I have a feeling these are not platforms, but are copied straight from the original template.......

Reply:
opened https://github.com/devcontainers/templates/issues/145 because I remembered you saying something like "the devcontainers official people get paid to do this" so I figured might as well ask, right? cant hurt.

Reply:
Thanks for opening that!
I too don't think this field is currently being used for anything.

So we can keep this as it is for now.